WooCommerce/Classes/ViewRelated/Dashboard/Free Trial/FreeTrialBannerPresenter.swift

```diff
@@ -103,7 +103,7 @@ private extension FreeTrialBannerPresenter {
         freeTrialBanner?.removeFromSuperview()
 
         let freeTrialViewController = FreeTrialBannerHostingViewController(mainText: contentText) { [weak self] in
-            self?.showUpgradePlanWebView()
+            self?.showSubscriptionsView()
```

Comment:
Since we have `freeTrialUpgrade` feature flag, Do you think we should have these actions and buttons to named according to the feature flag value?

Reply:
Do you mean even the ones inside the `HubMenu` or just the `showSubscriptionsView` action? Anyway, I adjusted it here: https://github.com/woocommerce/woocommerce-ios/pull/9627/commits/f736e4d35fd95fce1296b5b2cf61063d17094420